packages/react-components/react-progress/Spec.md

```diff
@@ -2,62 +2,105 @@
 
 ## Background
 
-_Description and use cases of this component_
+A `Progress` is a component that is used to display the current progress of an operation flow.
 
 ## Prior Art
 
-_Include background research done for this component_
+### Open UI
```

Comment:
I don't see a reference there to [OpenUI](https://open-ui.org/), is the title correct?

Reply:
@tomi-msft you can add a link to Open UI if there is a page about `Progress` there. If a page for progress does not exist feel free to rename this section to something like "Comparison between other libraries".